implementations/rust/ockam/ockam_command/src/project/util.rs

```diff
@@ -254,9 +257,35 @@ pub mod config {
         config.set_project_alias(
             project.name.to_string(),
             ProjectLookup {
-                node_route: proute,
+                node_route: Some(proute),
+                id: project.id.to_string(),
+                identity_id: Some(pid.clone()),
+                authority,
+            },
+        )?;
+        Ok(())
+    }
+
+    pub async fn set_project_unchecked(config: &OckamConfig, project: &Project<'_>) -> Result<()> {
+        let authority = if let Some(r) = &project.authority_access_route {
+            let rte = MultiAddr::try_from(&**r).context("Invalid project authority address")?;
+            let a = project
+                .authority_identity
+                .as_ref()
+                .context("Missing project authority")?;
+            let a = hex::decode(&**a).context("Invalid project authority")?;
+            let v = Vault::default();
+            let p = PublicIdentity::import(&a, &v).await?;
+            Some(ProjectAuthority::new(p.identifier().clone(), rte, a))
+        } else {
+            None
+        };
+        config.set_project_alias(
+            project.name.to_string(),
+            ProjectLookup {
+                node_route: None,
                 id: project.id.to_string(),
-                identity_id: pid.clone(),
+                identity_id: None,
```

Comment:
You should also pass the node_route and identity_id here based on the `Project`:

```rust
let node_route: Option<MultiAddr> = if project.access_route.is_empty() {
    Some(
        project
            .access_route
            .as_ref()
            .try_into()
            .context("Invalid project node route")?,
    )
} else {
    None
};

...

ProjectLookup {
    node_route,
    identity_id: project.identity_id.clone(),
    ...
}
```